src/components/List/HierarchyList/HierarchyList.jsx

```diff
@@ -36,6 +36,8 @@ const propTypes = {
   pageSize: PropTypes.string,
   /** Item id to be pre-selected */
   defaultSelectedId: PropTypes.string,
+  /** Item ids to be pre-expanded */
+  defaultExpandedIds: PropTypes.array,
```

Comment:
Checks are failing because linting is failing due to rule [`react/forbid-prop-types`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/forbid-prop-types.md). 

This needs to specify what type is in the array. I think they're strings, right?
```suggestion
  defaultExpandedIds: PropTypes.arrayOf(PropTypes.string),
```

Reply:
@tay1orjones , I made the suggested to the code and added a unit test.